packages/compass-saved-aggregations-queries/src/components/open-item-modal.tsx

```diff
@@ -161,6 +171,15 @@ const OpenItemModal: React.FunctionComponent<OpenItemModalProps> = ({
             label="Collection"
           ></CollectionSelect>
         </div>
+        <Checkbox
+          className={checkbox}
+          checked={updateItemNamespace}
+          onChange={(event) => {
+            dispatch(updateItemNamespaceChecked(event.target.checked));
```

Comment:
Super nit, feel free to ignore: as we are already using `connect` and `mapDispatch` here, it might be a bit cleaner to stick with the pattern for now